### PR TITLE
Constellar's Minors 1

### PR DESCRIPTION
**Pull request by Velius94**
## Changelog:
- Fixed Mar'alith dynasty by giving the commander a proper name and his missing spouse, who also happens to become an old god worshipper in 603+ bookmarks.
- Replaced a randomly generated Sunstrider duplicate dynasty by adding a lore-based Sunbright dynasty, whose member is a rogue named Asira.

# How to test:
- Check whether the characters show up in correct places and their religion/cult membership/traits are correctly assigned depending on time periods. The Mar'aliths reside in Silithus, while Asira should either be in Ravenholdt (583) and Quel'thalas (603)

### CODE REVIEW

**Comment by zumbak04** [comment]
@Velius94 don't label `suggestion - bla-bla-bla`. It's for suggestions in the **_Issues_**.

File: history/characters/42000_high_elf.txt [hunk 1]
```diff
@@ -1645,3 +1645,41 @@
...
+	culture=high_elf religion=cult_of_sunwell
+	martial=4 diplomacy=4 stewardship=5 intrigue=6 learning=5
+	add_trait=elusive_shadow add_trait=greedy add_trait=lustful add_trait=hedonist add_trait=proud
+	530.2.12={ birth=yes add_trait=creature_high_elf immortal_age=18 }
+	545.9.11={
+		add_trait = class_rogue_4
+	}
+	# Becomes an assassin in Ravenholdt
+	580.1.1={
+		employer = 58018
```
Comment (zumbak04) [inline history/characters/42000_high_elf.txt]: Error from `error.log`.
```
[characterhistory.cpp:1270]: Character history: Setting employer of Asira Sunbright ( 42308 ) to Jorach Raven ( 58018 ) who can't have a court
```

File: history/characters/340000_night_elf.txt [hunk 2]
```diff
@@ -1,16 +1,52 @@
...
+			effect={
+					add_lover=340001
+			}
+	}
 	1059.11.25={ death=yes }
 }
+340001={
+	name=Natalia
+	female=yes
+	dynasty=128000
```
Comment (zumbak04) [inline history/characters/340000_night_elf.txt]: She's from the husband's dynasty, so it looks like incest.
<details>
<summary>Click to expand</summary>

image

</details>

File: history/characters/340000_night_elf.txt [hunk 3]
```diff
@@ -1,16 +1,52 @@
...
 340000={
-	name=Mar'alith
+	name=Pas'draen
 	dynasty=128000
-	dna="kinkbpfiibj"
+	dna="kpekbpgnfeg"
 	culture=night_elf religion=kaldorei_religion
 	martial=5 diplomacy=6 stewardship=7 intrigue=3 learning=8
 	add_trait=skilled_tactician add_trait=diligent add_trait=brave add_trait=patient add_trait=strategist
 	disallow_random_traits = yes
```
Comment (zumbak04) [inline history/characters/340000_night_elf.txt]: If we don't have enough lore info, I would remove it to make other (traits you added above won't be touched) character traits random.

File: history/characters/340000_night_elf.txt [hunk 4]
```diff
@@ -1,16 +1,52 @@
...
 }
+340001={
+	name=Natalia
+	female=yes
+	dynasty=128000
+	dna="kinkbpfiibj"
+	culture=night_elf religion=kaldorei_religion
+	martial=4 diplomacy=5 stewardship=6 intrigue=5 learning=8
+	add_trait=scholarly_theologian add_trait=honest add_trait=shrewd add_trait=diligent add_trait=chaste
+	disallow_random_traits = yes
```
Comment (zumbak04) [inline history/characters/340000_night_elf.txt]: If we don't have enough lore info, I would remove it to make other (traits you added above won't be touched) character traits random.

File: history/characters/340000_night_elf.txt (hunk 2 above)
Comment (Velius94) [reply history/characters/340000_night_elf.txt]: can be resolved by making the commander marry into her dynasty matrilineally due to matriarchal society of the kaldorei

Comment (zumbak04) [reply history/characters/340000_night_elf.txt]: He needs a dynasty though. He's a feudal ruler.

Comment (Velius94) [reply history/characters/340000_night_elf.txt]: will just remove dynasty from Natalia instead

**Pull request merged**